packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,210 @@
+part of firebase_remote_config;
+
+/// LastFetchStatus defines the possible status values of the last fetch.
+enum LastFetchStatus { success, failure, throttled, noFetchYet }
+
+class FetchThrottledException implements Exception {
+  DateTime _throttleEnd;
+
+  FetchThrottledException._({int endTimeInMills = 43200}) {
+    _throttleEnd = new DateTime.fromMillisecondsSinceEpoch(endTimeInMills);
+  }
+
+  DateTime get throttleEnd => _throttleEnd;
+  String get msg {
+    final Duration duration = _throttleEnd.difference(new DateTime.now());
+    return '''Fetching throttled try again in ${duration.inMilliseconds}
+milliseconds''';
+  }
+
+  @override
+  String toString() {
+    final Duration duration = _throttleEnd.difference(new DateTime.now());
+    return '''FetchThrottledException
+Fetching throttled try again in ${duration.inMilliseconds} milliseconds''';
+  }
+}
+
+/// The entry point for accessing Remote Config.
+///
+/// You can get an instance by calling [RemoteConfig.instance]. Note
+/// [RemoteConfig.instance] is async.
+class RemoteConfig {
+  static const MethodChannel _channel =
+      const MethodChannel('plugins.flutter.io/firebase_remote_config');
+
+  static const String defaultValueForString = '';
+  static const int defaultValueForInt = 0;
+  static const double defaultValueForDouble = 0.0;
+  static const bool defaultValueForBool = false;
+
+  static const String fetchFailedThrottled = 'FETCH_FAILED_THROTTLED';
+
+  Map<String, RemoteConfigValue> _parameters;
+
+  DateTime _lastFetchTime;
+  LastFetchStatus _lastFetchStatus;
+  RemoteConfigSettings _remoteConfigSettings;
+
+  RemoteConfig._() {
+    _channel.setMethodCallHandler((MethodCall call) async {
+      switch (call.method) {
+        case 'UpdateFetch':
```

Comment:
I think you might be able to remove this

Reply:
Done